spec/pardot/objects/custom_fields_spec.rb

```diff
@@ -0,0 +1,58 @@
+require File.expand_path(File.dirname(__FILE__) + '/../../spec_helper')
+
+describe Pardot::Objects::CustomFields do
+  
+  before do
+    @client = create_client
+  end
+  
+  describe "query" do
```

Comment:
`"#query"`

Reply:
Yeah sadly that's the current idiom and I want to get this merged upstream so holding off on changes like that unless we decide tor really adopt it 100%